scripts/va-api-test.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+require 'jwt'
+require 'net/http'
+
+payload = { inherited_proofing_auth: 'mocked-auth-code-for-testing', exp: 1651943405 }
```

Comment:
maybe we can make this slightly more future-proofed?
```suggestion
payload = { inherited_proofing_auth: 'mocked-auth-code-for-testing', exp: 1.day.from_now.to_i }
```